nilearn/plotting/tests/test_img_plotting.py

```diff
@@ -566,6 +567,58 @@ def test_plot_connectome(tmpdir):
     plot_connectome(*args, node_color=['red'], display_mode='lzry')
     plt.close()
 
+    # Non symmetric matrix
+    adjacency_matrix = np.array([[1., -2., 0.3, 0.2],
+                                 [0.1, 1, 1.1, 0.1],
+                                 [0.01, 2.3, 1., 3.1],
+                                 [0.6, 0.03, 1.2, 1.]])
+    ax = plot_connectome(adjacency_matrix,
+                         node_coords,
+                         display_mode='ortho')
+    # No thresholding was performed, we should get
+    # as many arrows as we have edges
+    for direction in ['x', 'y', 'z']:
+        assert(len([patch for patch in ax.axes[direction].ax.patches
+             if isinstance(patch, FancyArrow)]) ==
+                    np.prod(adjacency_matrix.shape))
+
+    # Set a few elements of adjacency matrix to zero
+    adjacency_matrix[1, 0] = 0.0
+    adjacency_matrix[2, 3] = 0.0
+    # Plot with different display mode
+    ax = plot_connectome(adjacency_matrix,
+                         node_coords,
+                         display_mode='lzry')
+    # No edge in direction 'l' because of node coords
+    assert(len([patch for patch in ax.axes['l'].ax.patches
+             if isinstance(patch, FancyArrow)]) == 0)
+    for direction in ['z', 'r', 'y']:
+        assert(len([patch for patch in ax.axes[direction].ax.patches
+             if isinstance(patch, FancyArrow)]) ==
+                    np.prod(adjacency_matrix.shape) - 2)
```

Comment:
This is not related to this PR ? I'm confused.

Reply:
I am testing that we get the correct number of arrows in the connectome plot when setting some (2) elements of the matrix to zero and for a different display mode. If this is too redundant with other tests added in this PR, I can delete it

Reply:
OK, sounds right.